src/drawer.js

```diff
@@ -69,6 +69,7 @@ export default class Drawer extends util.Observer {
         if (this.params.fillParent || this.params.scrollParent) {
             this.style(this.wrapper, {
                 width: '100%',
+                cursor: this.params.hideCursor ? 'hidden' : 'auto',
```

Comment:
As per [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor), It seems it should be `none` instead of `hidden`?

Reply:
yes